src/SeleniumLibrary/__init__.py

```diff
@@ -431,6 +442,7 @@ def __init__(
         self,
         timeout=timedelta(seconds=5),
         implicit_wait=timedelta(seconds=0),
+        page_load_timeout=timedelta(seconds=10),
```

Comment:
@emanlove , should this be placed at the bottom for backwards compatibility reasons?